lib/emoji.js

```diff
@@ -20,6 +20,12 @@ module.exports = new Emoji();
 Emoji.parser = /:([a-zA-Z0-9_\-\+]+):/g;
 
 /**
+ * regex to parse emoji and string emoji in a string - finds emoji,
+ * e.g. :coffee:, <3, :), ;), :D
+ */
+Emoji.stringEmojiParser = /([;:<][()3D\/])/g;
```

Comment:
@omnidan We need to discuss this, currently it will replace all basics text emoji no matter what's before or after